### PR TITLE
FEAT: Added productlisting-filters

### PR DESCRIPTION
**Pull request by kotesh-arya**
## Added the following filters for ProductListing Page
- Rating filter 
- Men, Women, Stock, Delivery type filters.
- Price sorting

### CODE REVIEW

File: src/pages/ProductList/ProductlistCard.jsx [hunk 1]
```diff
@@ -16,16 +23,24 @@ function ProductlistCard({ title, imageSource, listedPrice, sellingPrice }) {
...
+        <img className="card-image" src={imageSource} alt="watch" />
       </div>
 
       <div className="card-content">
         <div className="card-title">{title}</div>
-
+        <div className="product-rating">
+          {Array(rating)
+            .fill()
+            .map((_, index) => (
```
Comment (swethaogeti) [inline src/pages/ProductList/ProductlistCard.jsx:34]: nice trick !!

File: src/pages/ProductList/filters/filters.jsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            <h4>Price</h4>
+            <li className="sorting-item">
+              <input
+                type="radio"
+                name="sorting"
+                id="decrease"
+                checked={sortBy && sortBy === "HIGH_TO_LOW"}
+                onChange={() => {
+                  dispatch({ type: "SORT", payload: "HIGH_TO_LOW" });
+                  // console.log("decrease");
```
Comment (swethaogeti) [inline src/pages/ProductList/filters/filters.jsx]: remove comments

**Review by swethaogeti — APPROVED** [review]
everything looks clean . Be consistent with z-index and margin . Rest is fine. GTM

**Review by MaruthiPothuganti — CHANGES_REQUESTED** [review]
A bug while toggling the women category filter in Product listing Page. Except that everything else looks good

image

image

File: src/pages/ProductList/ProductlistCard.jsx (hunk 1 above)
Comment (kotesh-arya) [reply src/pages/ProductList/ProductlistCard.jsx:34]: Thank you!!!

File: src/pages/ProductList/filters/filters.jsx (hunk 2 above)
Comment (kotesh-arya) [reply src/pages/ProductList/filters/filters.jsx]: Sure 👍🏻

**Comment by kotesh-arya** [comment]
> Fixed it and Thanks for pointing out 👍🏻 

>Screenshot (38)

**Pull request merged**